object-cache.php

```diff
@@ -354,68 +360,103 @@ function get( $id, $group = 'default', $force = false, &$found = null ) {
 		return $value;
 	}
 
-	function get_multi( $groups ) {
-		/*
-		format: $get['group-name'] = array( 'key1', 'key2' );
-		*/
+	function get_multiple( $ids, $group = 'default', $force = false ) {
+		$mc =& $this->get_mc( $group );
+
+		$no_mc = in_array( $group, $this->no_mc_groups );
```

Comment:
I like the approach of storing the result of the expression in a variable, as it makes the code more readable and also likely more performant 👍